src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html

```diff
@@ -15,6 +15,7 @@
      data-placement="left"
      [ngbTooltip]="usageTooltipTpl">
   <div class="progress-bar bg-info"
+       [ngClass]="{'bg-warning': usedPercentage > 90, 'bg-danger': usedPercentage === 100}"
```

Comment:
IMHO this is the way it should be done (no pipes, just styling in the HTML side). However, the fixed thresholds need to be replaced. And I don't understand how this values resulted in the screenshots you pasted, since these are different thresholds 🤔

Reply:
Done.